autometa/binning/recursive_dbscan.py

```diff
@@ -678,6 +685,27 @@ def main():
     )
     parser.add_argument("--purity", help="<purity cutoff>", default=90.0, type=float)
     parser.add_argument("--taxonomy", help="</path/to/taxonomy.tsv>")
+    parser.add_argument(
+        "--starting-rank",
+        help="Canonical rank at which to begin subsetting taxonomy",
+        default="superkingdom",
+        choices=[
+            "superkingdom",
+            "phylum",
+            "class",
+            "order",
+            "family",
+            "genus",
+            "species",
+        ],
+    )
+    parser.add_argument(
+        "--reverse-ranks",
+        action="store_true",
+        default=False,
+        help="Reverse canonical-rank taxonomy splitting"
+        " from superkingdom -> species to species -> superkingdom",
```

Comment:
I think this help message should more explicitly say what the order will be if `--reverse-ranks` is set to `True` vs. `False`, because as written, this will be confusing to users.